l3kernel/l3box.dtx

```diff
@@ -874,6 +874,68 @@
 %   dependent on the height of the original and \emph{vice versa}.
 % \end{function}
 %
+% \section{Viewing part of a box}
+%
+% \begin{function}[updated = 2023-04-14]
+%   {
+%     \box_set_clipped:N, \box_set_clipped:c,
+%     \box_gset_clipped:N, \box_gset_clipped:c
+%   }
+%   \begin{syntax}
+%     \cs{box_set_clipped:N} \meta{box}
+%   \end{syntax}
+%   Clips the \meta{box} in the output so that only material inside the
+%   bounding box is displayed in the output. The updated \meta{box} is an
+%   hbox, irrespective of the nature of the \meta{box} before the clipping is
+%   applied.
+%
+%   \begin{texnote}
+%     Clipping is implemented by the driver, and as such the full content of
+%     the box is placed in the output file. Thus clipping does not remove
+%     any information from the raw output, and hidden material can therefore
+%     be viewed by direct examination of the file.
+%   \end{texnote}
+% \end{function}
+%
+% \begin{function}[added = 2019-01-23]
+%   {
+%     \box_set_trim:Nnnnn, \box_set_trim:cnnnn,
+%     \box_gset_trim:Nnnnn, \box_gset_trim:cnnnn
+%   }
+%   \begin{syntax}
+%     \cs{box_set_trim:Nnnnn} \meta{box} \Arg{left} \Arg{bottom} \Arg{right} \Arg{top}
+%   \end{syntax}
+%   Adjusts the bounding box of the \meta{box} \meta{left} is removed from
+%   the left-hand edge of the bounding box, \meta{right} from the right-hand
+%   edge and so fourth. All adjustments are \meta{dim exprs}.
+%   Material outside of the bounding box is still displayed in the output
+%   unless \cs{box_clip:N} is subsequently applied.
+%   The updated \meta{box} is an
+%   hbox, irrespective of the nature of the \meta{box} before the trim
+%   operation is applied.
+%   The behavior of the operation where the trims requested is
+%   greater than the size of the box is undefined.
+% \end{function}
+%
+% \begin{function}[added = 2019-01-23]
+%   {
```

Comment:
shouldn't that give the current day? well, I guess there are arguments for both so not sure

Reply:
The code in the current form was added then to `l3candidates`: the move to stable doesn't affect that. So I thought this made the most sense: will be different for a move of `l3graphics`, for example, as that's not currently in `l3kernel`.

Reply:
that's why I said there are arguments both ways